server/src/main/resources/reference.conf

```diff
@@ -1,4 +1,4 @@
-# Nomad configuration
 broccoli {
   nomad.url = "http://localhost:4646"
+  templates.path = "templates"
```

Comment:
Should we perhaps use `./templates` to make clear that it's a path relative to the local directory?